app/views/pages/computers_for_kids_privacy_notice.html.erb

```diff
@@ -8,65 +8,66 @@
-  <li>the number and types of device in which the organisation has expressed an interest</li>
-</ul>
+    <p class="govuk-body">This notice explains how we use the information that you supply to us.</p>
 
-<h3 class="govuk-heading-m">How we use your personal information</h3>
+    <p class="govuk-body">When you opt in to the Computers for Kids scheme, the following non personal information is shared by DfE with Computacenter (UK) Limited (“Computacenter”).  Computacenter share this information with Mail Force:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>organisation details (name, address and Unique Registration Number, as published e.g. on <%= govuk_link_to 'Get Information About Schools', 'https://www.get-information-schools.service.gov.uk/' %>)</li>
+      <li>the number and types of device in which the organisation has expressed an interest</li>
+    </ul>
 
-<p class="govuk-body">We collect your name and contact details (email address and telephone number) associated with your organisation and share it with Computacenter, who use the information to:</p>
-<ul class="govuk-list govuk-list--bullet">
-  <li>co-ordinate allocation and delivery of devices to your organisation</li>
-</ul>
+    <h2 class="govuk-heading-m">How we use your personal information</h2>
 
-<p class="govuk-body">The DfE is the data controller for the DfE's processing of the personal information for the purposes of collecting and sharing with Computacenter your organisation’s expression of interest.  The DfE is not the data controller (or otherwise responsible for) Computacenter's processing of your personal information.</p>
+    <p class="govuk-body">We collect your name and contact details (email address and telephone number) associated with your organisation and share it with Computacenter, who use the information to:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>co-ordinate allocation and delivery of devices to your organisation</li>
+    </ul>
 
-<h3 class="govuk-heading-m">When we can use your personal information</h3>
-<p class="govuk-body">To use your personal information we need to meet a condition in the data protection legislation. For this service, we use your personal information as part of our legitimate interests (and the legitimate interests of Computacenter and Mail Force) to collect and share your opt-in information with Computacenter to allow Computacenter to co-ordinate the allocation and delivery of devices to your organisation under the Computers for Kids scheme.</p>
+    <p class="govuk-body">The DfE is the data controller for the DfE's processing of the personal information for the purposes of collecting and sharing with Computacenter your organisation’s expression of interest.  The DfE is not the data controller (or otherwise responsible for) Computacenter's processing of your personal information.</p>
```

Comment:
```suggestion
    <p class="govuk-body">The DfE is the data controller for the DfE’s processing of the personal information for the purposes of collecting and sharing with Computacenter your organisation’s expression of interest.  The DfE is not the data controller (or otherwise responsible for) Computacenter’s processing of your personal information.</p>
```